CMakeLists.txt

```diff
@@ -132,12 +132,17 @@ if (("${SEQAN_BUILD_SYSTEM}" STREQUAL "SEQAN_RELEASE") OR
     add_subdirectory (demos)
```

Comment:
remove first condition for demos. There is no SEQAN_RELEASE (anymore), only SEQAN_RELEASE_APPS and SEQAN_RELEASE_LIBRARY. The latter automatically gets the demos it rquires for dox.